src/NServiceBus.RabbitMQ/Connection/IClusterHostSelectionStrategy.cs

```diff
@@ -1,42 +0,0 @@
-namespace EasyNetQ
```

Comment:
@andreasohlund I've re-introduced the obsoleted interface in v3-obsoletes.cs. Do you think the PR is good to go?